sdk/identity/identity/src/msal/utils.ts

```diff
@@ -64,15 +66,15 @@ export function getAuthority(tenantId: string, host: string = DefaultAuthorityHo
 
 /**
  * Generates the known authorities.
- * If the tenantId is "adfs", we will return an array with the authorityHost as the only known authority.
- * Otherwise, it is safe to return an empty array.
  * @internal
  */
-export function getKnownAuthorities(tenantId: string, authorityHost: string): string[] {
-  if (tenantId === "adfs" && authorityHost) {
-    return [authorityHost];
-  }
-  return [];
+export function getKnownAuthorities(_tenantId: string, authorityHost: string): string[] {
+  // if (tenantId === "adfs" && authorityHost) {
+  //  return [authorityHost];
+  // }
+  // It seems safer to me to always specify the authority host.
+  // I'm keeping this function in case I change my mind, to have a simple way to alter this everywhere at once.
```

Comment:
Please delete commented code

Reply:
I'd rather not! I'm moving away from code I intentionally wrote. I believe in my decision, but having the reference can help me in the future if something unexpected happens. For example, if MSAL's default changes and ours stays the same, this could help me see why. Also, if we decide to do more logic around this method, I will be able to see what was the original intention there. It's pretty small.

Reply:
I see but it is hard to maintain commented code, you could instead factor into into a separate function and keep it around.

Reply:
Keeping around unused functions is worse in my perspective 🤔

Reply:
How about gists? and provide link to that? Not sure if this will be worse

Reply:
I would do gists for longer code 🤔

Reply:
Do we strip comments from our ESMs? I don't mind having commented out stuff in the source files as a way to remind ourselves of what paths we've already been down, but wouldn't mind if we didn't ship them.

I'm more concerned about the actual change here. We're now disregarding `tenantId` when it used to be required to be set to `"adfs"` to return anything. Also, what is the true utility of this function if it just returns an array with its second argument in it? The previous TSDoc comment was not very helpful, either. Why does this function behave this way?

Reply:
The idea is that for “adfs” tenants, the given authorityHost has to be in the knownAuthorities. Only in that case, at least for now, the known authorities can’t be implicitly derived. But we can just ignore that and always define the known authorities.

Because I am not aware of other scenarios in which the known authorities might be altered by circumstances, I’d like this method to remain, to make it easier to alter this behavior later.

Reply:
I investigated more!

What MSAL says for `knownAuthorities` is that it’s: An array of URIs that are known to be valid. Used in B2C scenarios.

What this means though, is that `knownAuthorities` disable authority validation. We set this in the case of ADFS because the authority can’t be validated there.

I’ve confirmed with Scott Schaab that we don’t want to disable validation in other cases.

I’ll undo my changes here.